pkg/agent/route/route_windows.go

```diff
@@ -48,6 +49,11 @@ func NewClient(hostGateway string, serviceCIDR *net.IPNet, encapMode config.Traf
 // Service LoadBalancing is provided by OpenFlow.
 func (c *Client) Initialize(nodeConfig *config.NodeConfig) error {
 	c.nodeConfig = nodeConfig
+	// Enable IP-Forwarding on the interface of OVS bridge, and the host networking stack can be used to set a correct
```

Comment:
Could you add more information to the comments, like how the SNAT packets will be forwarded from the bridge interface, and how it is sent out to physical interface?

Reply:
Reminder.

Reply:
Sure, will update.